src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const th = [...document.querySelectorAll('th')];
+const tr = [...document.querySelectorAll('tr')].slice(1, -1);
+const tbody = document.querySelector('tbody');
+
+function sort(e) {
+  if (e.target.tagName === 'TH') {
+    const item = e.target;
+    const indexForSotr = th.indexOf(item);
+    let sortElements;
+
+    if (indexForSotr !== 3) {
+      sortElements = tr.sort((a, b) =>
+        a.cells[indexForSotr].innerHTML.localeCompare(
+          b.cells[indexForSotr].innerHTML));
+    } else {
+      sortElements = tr.sort((a, b) =>
+        corectSalary(a.cells[indexForSotr].innerHTML) - corectSalary(
+          b.cells[indexForSotr].innerHTML));
+    }
+
+    tbody.innerHTML = '';
+
+    for (let i = 0; i < tr.length; i++) {
+      tbody.appendChild(sortElements[i]);
+    }
+  }
+}
+document.addEventListener('click', sort);
+
+function corectSalary(string) {
+  let result = '';
```

Comment:
```suggestion
  let salary = '';
```
Better provide more descriptive names than just "result"